### PR TITLE
Docs: Provide format for point in time restore parameter on the databases

### PR DESCRIPTION
**Pull request by mkjmdski**

### CODE REVIEW

File: website/docs/r/mariadb_server.html.markdown [hunk 1]
```diff
@@ -72,7 +72,7 @@ The following arguments are supported:
 
 * `public_network_access_enabled` - (Optional) Whether or not public network access is allowed for this server. Defaults to `true`.
 
-* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore`, specifies the point in time to restore from `creation_source_server_id`.
+* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore`, specifies the point in time to restore from `creation_source_server_id`. It should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g.: `2013-11-08T22:00:40Z`.
```
Comment (stephybun) [inline website/docs/r/mariadb_server.html.markdown:75]:
```suggestion
* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore`, specifies the point in time to restore from `creation_source_server_id`. It should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g. `2013-11-08T22:00:40Z`.
```

File: website/docs/r/mysql_server.html.markdown [hunk 2]
```diff
@@ -69,13 +69,13 @@ The following arguments are supported:
...
 
-* `identity` - (Optional) An `identity` block as defined below. 
+* `identity` - (Optional) An `identity` block as defined below.
 
 * `infrastructure_encryption_enabled` - (Optional) Whether or not infrastructure is encrypted for this server. Defaults to `false`. Changing this forces a new resource to be created.
 
 * `public_network_access_enabled` - (Optional) Whether or not public network access is allowed for this server. Defaults to `true`.
 
-* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore`, specifies the point in time to restore from `creation_source_server_id`.
+* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore`, specifies the point in time to restore from `creation_source_server_id`. It should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g.: `2013-11-08T22:00:40Z`.
```
Comment (stephybun) [inline website/docs/r/mysql_server.html.markdown:78]:
```suggestion
* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore`, specifies the point in time to restore from `creation_source_server_id`. It should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g. `2013-11-08T22:00:40Z`.
```

File: website/docs/r/postgresql_server.html.markdown [hunk 3]
```diff
@@ -80,19 +80,19 @@ The following arguments are supported:
 
 -> **NOTE:** `public_network_access_enabled` doesn't support PointInTimeRestore mode.
 
-* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore` the point in time to restore from `creation_source_server_id`. 
+* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore` the point in time to restore from `creation_source_server_id`. It should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g.: `2013-11-08T22:00:40Z`.
```
Comment (stephybun) [inline website/docs/r/postgresql_server.html.markdown:83]:
```suggestion
* `restore_point_in_time` - (Optional) When `create_mode` is `PointInTimeRestore` the point in time to restore from `creation_source_server_id`. It should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g. `2013-11-08T22:00:40Z`.
```

File: website/docs/r/sql_database.html.markdown [hunk 4]
```diff
@@ -82,7 +82,7 @@ The following arguments are supported:
 
 * `source_database_id` - (Optional) The URI of the source database if `create_mode` value is not `Default`.
 
-* `restore_point_in_time` - (Optional) The point in time for the restore. Only applies if `create_mode` is `PointInTimeRestore` e.g. 2013-11-08T22:00:40Z
+* `restore_point_in_time` - (Optional) The point in time for the restore. Only applies if `create_mode` is `PointInTimeRestore`, it should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g.: `2013-11-08T22:00:40Z`.
```
Comment (stephybun) [inline website/docs/r/sql_database.html.markdown:85]:
```suggestion
* `restore_point_in_time` - (Optional) The point in time for the restore. Only applies if `create_mode` is `PointInTimeRestore`, it should be provided in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) format, e.g. `2013-11-08T22:00:40Z`.
```

**Review by stephybun — CHANGES_REQUESTED** [review]
Thanks amending these with an example @mkjmdski, I left some corrections on the punctuation but once that's fixed this will be good to merge!

**Comment by mkjmdski** [comment]
Hi @stephybun I've commited your suggestions, please feel free to merge this PR whenever you are ready :)

**Pull request merged**